
project build/soong/
diff --git a/ui/build/paths/config.go b/ui/build/paths/config.go
index ed382a5b..77feaac3 100644
--- a/ui/build/paths/config.go
+++ b/ui/build/paths/config.go
@@ -96,6 +96,8 @@ var Configuration = map[string]PathConfig{
 	"tr":      Allowed,
 	"unzip":   Allowed,
 	"zip":     Allowed,
+	"arm-linux-androidkernel-as":         Allowed,
+	"arm-linux-androidkernel-ld":         Allowed,
 
 	// Host toolchain is removed. In-tree toolchain should be used instead.
 	// GCC also can't find cc1 with this implementation.
